testutil-server/src/main/java/io/spine/testing/server/EmittedMessageSubject.java

```diff
@@ -102,7 +102,7 @@ public final void isNotEmpty() {
     }
 
     private IterableSubject assertActual() {
-        return check().that(actual());
+        return check("actual()").that(actual());
```

Comment:
This is of a little help to a developer who's not familiar with the `Subject`'s internal wiring.